cmd/queue/main.go

```diff
@@ -214,8 +215,44 @@ func createAdminHandlers() *http.ServeMux {
 	return mux
 }
 
+func execHTTPAction(url string) error {
+	resp, err := http.Get(url)
+	if err != nil {
+		return err
+	}
+
+	if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusBadRequest {
+		return fmt.Errorf("HTTP request failed with status: %v", resp.StatusCode)
+	}
+
+	return nil
+}
+
+func handleActions() (handled bool, err error) {
+	switch flag.Arg(0) {
+	case queue.CommandModeExec:
+		handled = true
+
+		if len(flag.Args()) > 1 {
+			switch flag.Arg(1) {
+			case queue.ActionQuit:
```

Comment:
So I think a better code would be if you have
1. map[string]string (action to path map)
2. then here: `if path, ok := map[action]; ok { return true, execHTTPAction..} else {return false /*we didn't handle, did we?*/, errors.New()}

Reply:
It feels like the concern was the duplication of generating the url which may have human errors when updating. I've extracted that out and now uses a map for finding the url for the specific action.